src/github/action/main.ts

```diff
@@ -55,15 +55,20 @@ const main = async () => {
 
   // Otherwise, check it through the API
   const checkReviewsApi = getInput("checks-reviews-api", { required: true });
-  const logLines = (await (
-    await fetch(checkReviewsApi, {
+
+  // cleaned up code into steps
+  try {
+    const externalAction = await fetch(checkReviewsApi, {
       method: "POST",
       headers: { "Content-Type": "application/json" },
       body: JSON.stringify(actionData),
-    })
-  ).json()) as string[];
-
-  logger.log(logLines.join("").trim());
+    });
+    const logLines = (await externalAction.json()) as string[];
+    debug(JSON.stringify(logLines));
+    logger.log(logLines.join("").trim());
+  } catch (e) {
+    setFailed(e);
```

Comment:
wanted to log debug/log here too? or that will be output by setFailed?

Reply:
This will be output by set failed, and we can also run the `debug` command to see more in-detail information.